HTTP_200/urls.py

```diff
@@ -24,6 +24,8 @@
     # url(r'^api/', include('feeds.urls')),
     # url(r'', include('rest_framework.urls', namespace='rest_framework'))
     url(r'^students/create/$', BulkUser.as_view(), name='bulk_students_create'),
+    url(r'^students/create/single/$', SingleUser.as_view(), name='single_user_create'),
```

Comment:
line 26 and 27 should be interchanged and line 28 needs to be removed(blank line)